src/main.ts

```diff
@@ -369,14 +370,56 @@ async function getServerPath(
 	}
 
 	const workspaceDependency = await getWorkspaceDependency(outputChannel);
+	if (workspaceDependency) {
+		return {
+			bundled: false,
+			workspaceDependency: true,
+			command: workspaceDependency,
+		};
+	}
+
+	if (config.get<boolean | undefined>("biome.searchInPath", true) === true) {
+		outputChannel.appendLine("Searching for Biome in PATH");
+		const biomeInPATH = await findBiomeInPath();
+		if (biomeInPATH) {
+			outputChannel.appendLine(`Biome found in PATH: ${biomeInPATH.fsPath}`);
+			return {
+				bundled: false,
+				workspaceDependency: false,
+				command: biomeInPATH.fsPath,
+			};
+		}
+	}
+
+	// Last resort
 	return {
-		bundled: workspaceDependency === undefined,
-		workspaceDependency: workspaceDependency !== undefined,
-		command:
-			workspaceDependency ?? (await getBundledBinary(context, outputChannel)),
+		bundled: true,
+		workspaceDependency: undefined,
+		command: await getBundledBinary(context, outputChannel),
 	};
 }
 
+/**
+ * Attempts top resolve the path to the biome binary from the PATH environment variable.
+ *
+ * We manually scan all the folders in the path because we may not always have access to
+ * `which` or `where` on the system, or in the PATH.
+ */
+async function findBiomeInPath(): Promise<Uri | undefined> {
+	const path = process.env.PATH;
+
+	if (!path) {
+		return;
+	}
+
+	for (const dir of path.split(delimiter)) {
```

Comment:
Shouldn't we split by `:`? `path.delimiter` is `/` or `\`, and `$PATH` uses `:` to delimit the paths.

Reply:
`:` is a path _delimiter_
`/` is a path _separator_

https://nodejs.org/api/path.html#pathdelimiter
https://nodejs.org/api/path.html#pathsep

Reply:
Oooh, I missed that. Thank you!